src/ripple/app/consensus/RCLValidations.cpp

```diff
@@ -191,13 +193,30 @@ handleNewValidation(Application& app,
     if (masterKey)
     {
         ValStatus const outcome = validations.add(calcNodeID(*masterKey), val);
+
         if(j.debug())
             dmp(j.debug(), to_string(outcome));
 
-        if (outcome == ValStatus::badSeq && j.warn())
+        if (outcome == ValStatus::conflicting && j.fatal())
```

Comment:
Note that this checks `j.fatal()`, but logs below with `j.warn()`.  Same for line 208